src/main/java/seedu/financialplanner/enumerations/ExpenseType.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Arrays;
 
 public enum ExpenseType {
-    DINING, ENTERTAINMENT, SHOPPING, TRAVEL, INSURANCE, NECESSITIES, OTHERS;
+    DINING, ENTERTAINMENT, SHOPPING, TRAVEL, INSURANCE, NECESSITIES, OTHERS, GOAL;
```

Comment:
can the goal be one of the expensetypes? ie. buy car would be under 'others'. so goal shouldnt be one of the types

Reply:
Make sense.